evm/src/main/java/org/hyperledger/besu/evm/worldstate/UpdateTrackingAccount.java

```diff
@@ -74,6 +75,7 @@
 
     this.updatedCode = Bytes.EMPTY;
     this.updatedStorage = new TreeMap<>();
+    this.updatedTransientStorage = new TreeMap<>();
```

Comment:
Why use a TreeMap here?

Reply:
I was simply basing it off of regular storage, but I think you're right that it's not necessary to use a `TreeMap` since its keys are not iterated through in a particular order.